git-p4.py

```diff
@@ -1257,9 +1257,13 @@ def generatePointer(self, contentFile):
             pointerFile = re.sub(r'Git LFS pointer for.*\n\n', '', pointerFile)
```

Comment:
First at all - thank you for your comments

> The reason why you ended up with this unreadably long subject line
> is because your commit did not follow the recommended

Yep, this is my mistake - i thought the subject of the letter is  "git-p4: use lfs.storage instead of local .git/lfs"But in https://public-inbox.org/git/pull.483.git.1575466209.gitgitgadget@gmail.com/#t  i see the next email subject:"Subject: [PATCH 0/1] git-p4: use lfs.storage instead of local .git/lfs" and "Subject: [PATCH 1/1] git-p4: use lfs.storage instead of local .git/lfs Use lfs.storage if it defined in git.config. If lfs.storage not define - used local .git/lfs. Original code uses local .git/lfs in sync/clone operations, but if you have external lfs storage better to use it." - so i little confused how that works.

> What is "git.config"?  Is it our own usual configuration file(s), or
> is it the Git-LFS customized .lfsconfig file at the root of the
> repository, or something else?

 git.config is the reference to [*1*] and in .lfsconfig we doesn`t have configuration for some kind lfs storage, as writed in [*2*], so it`s config file, which called via `cmd = [ "git", "config" ]` - it can be global or local .gitconfig file.
> Possible rewrite 
Yes, that`s make sense. Thanks.
>  The reason why I suspect so is because [*1*] says that the value of
Thank you for your comment - it`s more correct to use in that way
[Reference]

*1* https://github.com/git/git/commit/692e17964dd4f8da4b3994c5c7055ec9a8c22a75*2* https://github.com/git-lfs/git-lfs/blob/master/docs/man/git-lfs-config.5.ronn
*2* https://github.com/git-lfs/git-lfs/blob/master/docs/man/git-lfs-config.5.ronn

Reply:
/preview

Reply:
@panzercheg first of all, Junio cannot see your replies here, you really need to send them as mail, as indicated in the [reply to this](https://github.com/gitgitgadget/gitgitgadget/wiki/ReplyToThis) link at the top of the comment you replied to.

Also, the `/preview` command needs to be in a top-level comment, not in a code comment thread.

Reply:
ohh, got it. Thank you.